app/bundles/EmailBundle/Controller/AjaxController.php

```diff
@@ -228,6 +228,10 @@ protected function testEmailServerConnectionAction(Request $request)
                             $mailer->setHost($settings['amazon_region']);
                         }
                     }
+
+                    if ('mautic.transport.sparkpost' == $transport) {
```

Comment:
Is this a typo?    I don't really know, but It says **sparkpost** but this is a mailjet pull request.